src/libraries/Microsoft.VisualBasic.Core/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -1,28 +1,4 @@
-MembersMustExist : Member 'Microsoft.VisualBasic.CompilerServices.NewLateBinding.FallbackInvokeDefault1(System.Object, System.Object[], System.String[], System.Boolean)' does not exist in the reference but it does exist in the implementation.
-MembersMustExist : Member 'Microsoft.VisualBasic.CompilerServices.NewLateBinding.FallbackInvokeDefault2(System.Object, System.Object[], System.String[], System.Boolean)' does not exist in the reference but it does exist in the implementation.
-MembersMustExist : Member 'Microsoft.VisualBasic.CompilerServices.NewLateBinding.FallbackSet(System.Object, System.String, System.Object[])' does not exist in the reference but it does exist in the implementation.
-MembersMustExist : Member 'Microsoft.VisualBasic.CompilerServices.NewLateBinding.FallbackSetComplex(System.Object, System.String, System.Object[], System.Boolean, System.Boolean)' does not exist in the reference but it does exist in the implementation.
-MembersMustExist : Member 'Microsoft.VisualBasic.CompilerServices.NewLateBinding.LateCallInvokeDefault(System.Object, System.Object[], System.String[], System.Boolean)' does not exist in the reference but it does exist in the implementation.
-MembersMustExist : Member 'Microsoft.VisualBasic.CompilerServices.NewLateBinding.LateGetInvokeDefault(System.Object, System.Object[], System.String[], System.Boolean)' does not exist in the reference but it does exist in the implementation.
-MembersMustExist : Member 'Microsoft.VisualBasic.CompilerServices.Operators.FallbackInvokeUserDefinedOperator(System.Object, System.Object[])' does not exist in the reference but it does exist in the implementation.
-TypesMustExist : Type 'Microsoft.VisualBasic.CompilerServices.SiteDelegate0' does not exist in the reference but it does exist in the implementation.
-TypesMustExist : Type 'Microsoft.VisualBasic.CompilerServices.SiteDelegate1' does not exist in the reference but it does exist in the implementation.
-TypesMustExist : Type 'Microsoft.VisualBasic.CompilerServices.SiteDelegate2' does not exist in the reference but it does exist in the implementation.
-TypesMustExist : Type 'Microsoft.VisualBasic.CompilerServices.SiteDelegate3' does not exist in the reference but it does exist in the implementation.
-TypesMustExist : Type 'Microsoft.VisualBasic.CompilerServices.SiteDelegate4' does not exist in the reference but it does exist in the implementation.
-TypesMustExist : Type 'Microsoft.VisualBasic.CompilerServices.SiteDelegate5' does not exist in the reference but it does exist in the implementation.
-TypesMustExist : Type 'Microsoft.VisualBasic.CompilerServices.SiteDelegate6' does not exist in the reference but it does exist in the implementation.
-TypesMustExist : Type 'Microsoft.VisualBasic.CompilerServices.SiteDelegate7' does not exist in the reference but it does exist in the implementation.
-
-# C# doesn't permit setting the parameter name of the setter.
```

Comment:
Does this comment still add value? If so, it should be kept.

Reply:
Not really sure why this was here but the whole purpose of these files is for them to be auto generated, and we were supposed to regenerate them every now and then (which clearly we haven't done in a while) so I wouldn't personally revert this change as comment will be removed next regeneration anyway.